recipe/meta.yaml

```diff
@@ -65,14 +64,14 @@ requirements:
     - setuptools
     - cudnn                                  # [cuda_compiler_version != "None"]
     - numpy
-    - pytorch
```

Comment:
Am guessing this was here to make our pinning logic work

Reply:
Yes

Reply:
It's not contained in the torchvision feedstock as far as I can see. Is there a reason for the inconsistency?

Reply:
Yeah. It uses a fixed pinning from upstream